tests/rkt_run_pod_manifest_test.go

```diff
@@ -367,6 +598,32 @@ func TestPodManifest(t *testing.T) {
 			"",
 		},
 		{
+			// Simple read after write with volume mounted in a read-only rootfs, no apps in pod manifest.
```

Comment:
I don't understand this one. There is no mount point, so the volume is not mounted and the test writes directly in the read-only rootfs. Why does it pass?